test/Raven/Tests/ClientTest.php

```diff
@@ -1281,6 +1281,36 @@ public function testSanitizeContexts()
         )), $data);
     }
 
+    /**
+     * @covers Raven_Client::sanitize
```

Comment:
There is no need to specify that the `sanitize` method is covered by this unit test